packages/react-composites/tests/app/call/BaseApp.tsx

```diff
@@ -46,7 +46,9 @@ export function BaseApp(props: { queryArgs: QueryArgs; callAdapter?: CallAdapter
                 queryArgs.injectParticipantMenuItems ? onFetchParticipantMenuItems : undefined
               }
               options={
-                queryArgs.injectCustomButtons
+                queryArgs.customCallCompositeOptions
+                  ? queryArgs.customCallCompositeOptions
+                  : queryArgs.injectCustomButtons
```

Comment:
Not part of this PR but can we remove `injectCustomButtons` query arg. It seems like a less flexible version of `customCallCompositeOptions`.